### PR TITLE
Feature/adding rubocop performance

### PR DESCRIPTION
**Pull request by andyduong1920**
## What happened
Solved https://github.com/nimblehq/rails-templates/issues/124

 
## Insight
Adding [rubocop-performance](https://github.com/rubocop-hq/rubocop-performance)

## Proof Of Work
The rubocop-performance is load 

image

### CODE REVIEW

File: .template/variants/web/spec/support/capybara.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@
   options.add_argument('no-sandbox')
 
   # Run headless by default unless CHROME_HEADLESS specified
-  options.add_argument('headless') unless ENV['CHROME_HEADLESS'] =~ /^(false|no|0)$/i
+  options.add_argument('headless') unless /^(false|no|0)$/i.match?(ENV['CHROME_HEADLESS'])
```
Comment (gutakk) [inline .template/variants/web/spec/support/capybara.rb:30]: Could you please explain more why you have to make this change?

I tried to add `rubocop-performance` to `Gemfile.tt` and require it in `.rubocop.yml` and create new rails app using this template.
After I run `rubocop --requrie rubocop-performance` there are only `FrozenStringLiteralComment` appear.

Thanks

Comment (andyduong1920) [reply .template/variants/web/spec/support/capybara.rb:30]: `rubocop-performance` warns this line @gutakk

Comment (andyduong1920) [reply .template/variants/web/spec/support/capybara.rb:30]: image

Comment (gutakk) [reply .template/variants/web/spec/support/capybara.rb:30]: thanks for explanation.

**Review by gutakk — APPROVED** [review]

**Review by olivierobert — APPROVED** [review]

**Pull request merged**